benchmarks/bench_mnist.py

```diff
@@ -89,6 +90,10 @@ def load_data(dtype=np.float32, order='F'):
     'SampledRBF-SVM':
     make_pipeline(RBFSampler(gamma=0.015, n_components=1000), LinearSVC(C=100)),
     'LinearRegression-SAG': LogisticRegression(solver='sag', tol=1e-1, C=1e4)
+    'MultilayerPerceptron': MLPClassifier(
```

Comment:
I updated `bench_mnist.py` with adam added and I reran all the models. 

`learning_rate_init=0.5` for sgd somehow fails to converge when I ran it, so I changed it to 0.2